app/views/shared/summary_cards/_resource.html.slim

```diff
@@ -55,7 +55,7 @@
       .summary-card__more.summary-card__more--meta
         .summary-card__metadata.summary-card__metadata--last
```

Comment:
Can we actually add a check here to see if there is a URL available at all? If there is no URL, there's no way for us to have any annotations, so this component wouldn't make sense to include.